app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -83,9 +83,7 @@ EOF
     zypper --quiet --non-interactive addrepo /tmp/foreman_registration.repo
   fi
 elif [ -f /etc/debian_version ]; then
-  cat << EOF > /etc/apt/sources.list.d/foreman_registration.list
-<%= shell_escape @repo %>
-EOF
+<%= save_to_file('/etc/apt/sources.list.d/foreman_registration.list', @repo) %>
```

Comment:
Maybe this should be indented. Not entirely sure. I wonder if we have a Debian-based snapshot to see the preview.
```suggestion
  <%= save_to_file('/etc/apt/sources.list.d/foreman_registration.list', @repo) %>
```